### ISSUE TITLE
'await operator' precedence

### ISSUE THREAD

**Issue by socketpair** [babel/babel#1225]
Traceur and babel understand that differently (!) who is right?
I think, code for both functions should be the same (as in traceur), but in babel, results are different (!).

```
async function asd() {
  return await qwe() == 6;
}

async function asd2() {
  return (await qwe()) == 6;
}
```

**Comment by sebmck** [comment]
```javascript
async function asd() {
  return await qwe() == 6;
}

async function asd2() {
  return (await qwe()) == 6;
}
```

doesn't even parse in Traceur so I'm not entirely sure what the issue is, can you elaborate?

**Comment by socketpair** [comment]
You forgot to turn on 'experimental' in traceur

**Comment by socketpair** [comment]
First, I did not red docs about precedence of that operator, so I don't known who is wrong -- babel or traceur.

**Comment by socketpair** [comment]
Also found that: https://gist.github.com/lukehoban/8678463

**Comment by socketpair** [comment]
WOW, Cool! Could you point to official ES7 documentation where this is specified?

**Comment by sebmck** [comment]
https://github.com/lukehoban/ecmascript-asyncawait#syntax

```
UnaryExpression :
    ...
    await [Lexical goal InputElementRegExp] UnaryExpression
```

**Comment by sebmck** [comment]
Sorry for the delay. Fixed as of 5.1.0, thanks!
